azuredevops/internal/acceptancetests/resource_check_branch_control_test.go

```diff
@@ -31,6 +31,7 @@ func TestAccCheckBranchControl_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(tfCheckNode, "project_id"),
 					resource.TestCheckResourceAttr(tfCheckNode, "allowed_branches", branches),
 					resource.TestCheckResourceAttr(tfCheckNode, "display_name", checkName),
+					resource.TestCheckResourceAttr(tfCheckNode, "timeout", "50000"),
```

Comment:
default value is `1440` not `5000`

Reply:
What about the values that are used in the Acceptance tests for "Check Approval"?
See:
https://github.com/microsoft/terraform-provider-azuredevops/blob/bd2d69d460e62ea22f0d11252d58268bd5c16130/azuredevops/internal/acceptancetests/resource_check_approval_test.go#L33

https://github.com/microsoft/terraform-provider-azuredevops/blob/bd2d69d460e62ea22f0d11252d58268bd5c16130/azuredevops/internal/acceptancetests/resource_check_approval_test.go#L59